.github/workflows/docker-publish.yml

```diff
@@ -95,6 +95,50 @@ jobs:
             devitocodes/devito:gpu-latest
             devitocodes/devito:gpu-${{ github.event.release.tag_name }}
 
+  deploy-docker-intel:
```

Comment:
Can't this be easily be just added as a different case in a `matrix` rather than duplicate all these steps just for a dockerfile name?

Reply:
agree